book/src/appendices/b_migration_notes/transform_api_changes.md

```diff
@@ -0,0 +1,175 @@
+# `Transform` API Changes
+
+The names of several `Transform` methods have been changed in order to better reflect their actual function and reduce potential semantic confusion. Many of these changes are breaking and old projects will have to be updated.
+
+## Summary
+
+
+* `Float` type has been added.
+* `GlobalTransform` has been removed.
+* `set` methods have been renamed using `set_translation`
+* `local` methods have been renamed using `append`
+* `pitch`, `yaw`, and `roll` methods now use x, y, and z axis respectively.
+* `global` methods have been renamed using `prepend`
+* `set_position` has been renamed `set_translation`
+* 2D helper aliases have been added
+* `set_rotation` methods have been added
+* `euler_angles` has been added
+
+## `Float` Type
+
+A wrapper type around `f32` and `f64`. It is used to hide the actual type being used internally. Mostly used with the Transform type. The default type is `f32` and you can switch to the `f64` type by enabling the "float64" feature gate.
+
+## Renamed Transform Methods (Breaking Changes)
+
+### `set(x|y|z)` to `set_translation(x|y|z)`
+
+```patch
+-transform.set_x(0.2);
++transform.set_translation_x(0.2);
+```
+
+```patch
+-transform.set_y(7.1);
++transform.set_translation_y(7.1);
+```
+
+```patch
+-transform.set_z(2.4);
++transform.set_translation_z(2.4);
+```
+
+```patch
+-transform.set_xyz(0.2, 1.0, 0.8);
++transform.set_translation_xyz(0.2, 1.0, 0.8);
+```
+
+### `_local` to `append_`
```

Comment:
Not exactly specific to this PR, but why was this changed? The previous naming seems more obvious to me.

Reply:
I can't comment on that, here's the relevant issue: https://github.com/amethyst/amethyst/pull/1451

Reply:
I see, thanks.